packages/flutter_tools/lib/src/commands/create.dart

```diff
@@ -4,6 +4,8 @@
 
 import 'dart:async';
 
+import 'package:linter/src/rules/pub/package_names.dart' as package_names; // ignore: implementation_imports
```

Comment:
Please file a bug with the linter package to request that this be exposed so that we don't have to cheat.